frontend/components/Files.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="wrapper">
+    <h1 class="title text-md lg:text-2xl 2xl:text-3xl">Pliki do pobrania</h1>
+    <div class="flow-root mt-6" id="files">
+      <ul class="-my-5 divide-y divide-gray-200">
+        <li v-for="document in documents" :key="document.handle" class="py-4">
+          <div class="flex items-center space-x-4">
+            <div class="flex-1 min-w-0">
+              <p
+                class="
+                  text-sm
+                  font-medium
+                  lg:text-xl
+                  2xl:text-2xl
+                  text-gray-900
+                  truncate
+                "
+              >
+                {{ document.name }}
+              </p>
+            </div>
+            <div>
+              <a
+                class="
+                  noselect
+                  btn
+                  inline-flex
+                  items-center
+                  shadow-sm
+                  px-2.5
+                  py-0.5
+                  border border-gray-300
+                  text-sm
+                  leading-5
+                  font-medium
+                  rounded-full
+                  text-gray-700
+                  bg-white
+                  hover:bg-gray-100
+                  xl:text-xl
+                "
+              >
+                Pobierz
+              </a>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+const documents = [
+  {
+    name: "Podanie o praktyki",
+    handle: "podanie",
+  },
+  {
+    name: "Zaświadczenie",
+    handle: "zaswiad",
+  },
+  {
+    name: "Umowa",
+    handle: "umowa",
+  },
+  { name: "Regulamin", handle: "regulamin" },
+];
+
+export default {
+  setup() {
+    return {
+      documents,
+    };
+  },
+};
+</script>
```

Comment:
1. Why are you mixing `Composition API` with `Options API`. Almost whole project is using `Options API`. Mixing them together is not that bad, when you for example refactor big project and don't have time to rewrite all components at once, but in this case you should establish one way.
2. Move documents to `reactive`.

Reply:
It was just for testing, we will get data from JSON later so it will be deleted.

Reply:
But I changed this to Options API.